universal_transfer_operator/src/constants.py

```diff
@@ -0,0 +1,78 @@
+import sys
+from enum import Enum
+
+# typing.Literal was only introduced in Python 3.8, and we support Python 3.7
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+
+class FileLocation(Enum):
+    # [START filelocation]
+    LOCAL = "local"
+    HTTP = "http"
+    HTTPS = "https"
+    GS = "gs"  # Google Cloud Storage
+    GOOGLE_DRIVE = "gdrive"
+    S3 = "s3"  # Amazon S3
+    WASB = "wasb"  # Azure Blob Storage
+    WASBS = "wasbs"  # Azure Blob Storage
+    # [END filelocation]
+
+    def __str__(self) -> str:
+        return self.value
+
+
+class IngestorSupported(Enum):
+    # [START transferingestor]
+    Fivetran = "fivetran"
+    # [END transferingestor]
+
+    def __str__(self) -> str:
+        return self.value
+
+
+class TransferMode(Enum):
+    # [START TransferMode]
+    NATIVE = "native"
+    NONNATIVE = "nonnative"
+    THIRDPARTY = "thirdparty"
+    # [END TransferMode]
+
+    def __str__(self) -> str:
```

Comment:
@sunank200 Can we implement __repr__ instead?

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1492/commits/ac3af739b548c86636c4519e0c1c7bcadf321ead)